.github/workflows/node.js.yml

```diff
@@ -21,6 +21,11 @@ jobs:
             VSCODE_TEST_VERSION: ${{ matrix.vscode-version }}
             NODE_OPTIONS: '--max-old-space-size=8192'
         steps:
+            - name: Install git-secrets
+              run: |
+                  brew update
+                  brew install git-secrets
+                  brew upgrade
```

Comment:
brew update/upgrade can be really slow, so this is a high time-cost in our CI just for git secrets :) 

isn't it just a script? can we `curl` the raw script perhaps?